templates/full/src/controls/ui.tsx

```diff
@@ -125,6 +125,53 @@ function QRCodeExample() {
   )
 }
 
+function ReactNativeExample() {
+  const data = [
+    {
+      key: 'rise-tools',
+      label: (
+        <Button theme="blue" marginVertical="$2">
+          Rise Tools
+        </Button>
+      ),
+    },
+    {
+      key: 'rise-tools-1',
+      label: (
+        <Button theme="blue" marginVertical="$2">
+          Rise Tools
+        </Button>
+      ),
+    },
+    {
+      key: 'rise-tools-2',
+      label: (
+        <Button theme="blue" marginVertical="$2">
+          Rise Tools
+        </Button>
+      ),
+    },
+  ]
+  return (
+    <>
+      <StackScreen title="QR Code" />
+      <YStack gap="$8" padding="$4">
```

Comment:
if we are putting all RN examples here in one place, we are missing a lot of things. I think at least the lists should be separate so we can have a list view here and show more of the components

Reply:
Ok, I'll revert List back to be top-level, I think rest we can showcase later when we have better idea how to do it! (kinda something else on my mind now)